src/zimg/colorspace/gamma.cpp

```diff
@@ -120,6 +120,16 @@ float rec_1886_inverse_eotf(float x) noexcept
 	return x < 0.0f ? 0.0f : zimg_x_powf(x, 1.0f / 2.4f);
 }
 
+float log100_oetf(float x) noexcept
+{
+	return x < 0.01f ? 0 : 1.0f + zimg_x_logf(x) / 2.0f;
```

Comment:
Isn't logf the base-2 logarithm? Based on the threshold of 0.01, it seems that the base-10 log10f needs to be used.

Reply:
hmm logf should be natural log, you're right log10 should be used, thanks for spotting